README.md

```diff
@@ -21,6 +21,23 @@ application is sufficient to make the module available to your application.
 
 TODO: -->
 
+## Description
+
+ -Module provies wrapper to a credentials store (bedrock-credentials-mongodb for example).
+
+ -Angular module provides directive for credentials, and provides routes that map to templates that have a controller that will
+ either store or transmit the credentials using navigator.credentials.transmit to a credential consumer.
+
+ -Can compose a view of an identity.
+
+## Dependencies:
```

Comment:
If you could link to the github repositories for "dependencies", that would be good. Also, I think we may want to change this to "Bedrock Dependencies", as there are lots more dependencies that a package has that may not be that important to list out here.